include/sdf/Plugin.hh

```diff
@@ -107,6 +119,14 @@ namespace sdf
     /// \param[in] _elem Element to insert.
     public: void InsertContent(const sdf::ElementPtr _elem);
 
+    /// \brief Insert an element into the plugin content. This does not
+    /// modify the values in the sdf::ElementPtr returned by the `Element()`
+    /// function.
+    /// \param[in] _elem Element to insert.
+    /// \param[out] _errors Vector of errors.
+    public: void InsertContent(const sdf::ElementPtr _elem,
```

Comment:
Can we make the output param come first for consistency?

Reply:
Sure, I also did it for the other case of `InsertContent`.